deployer/README.md

```diff
@@ -1067,11 +1066,7 @@ Deployment failure can happen for a number of transitory reasons, such as
 the image pull taking too long, or nodes being unresponsive. Examine the
 deployer pod logs for possible reasons; but often you can simply redeploy:
 
-    $ oc deploy --latest logging-es-2e7ut0iq
-
-Or you may be able to scale up the existing deployment:
```

Comment:
I've removed this suggestion since the RC at this point is marked as Failed. Scaling it up does not work anymore, I would expect, because of the use of informers in the DC controller (since 3.3 or 3.4, can never remember off the top of my head).